src/scripts/main.js

```diff
@@ -1,7 +1,31 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const messageBlock = document.createElement('div');
+
+  messageBlock.classList.add('notification');
+  messageBlock.classList.add(type);
+
+  const header = document.createElement('h2');
+
+  header.classList.add('title');
+  header.textContent = title;
+
+  const paragrath = document.createElement('p');
```

Comment:
There is a typo in the variable name 'paragrath'. It should be 'paragraph'. This typo will cause a reference error when trying to use this variable.